src/prototype/dom/layout.js

```diff
@@ -1677,27 +1530,6 @@
      !Element.descendantOf(element, document.body);
   }
 
-  // If the browser supports the nonstandard `getBoundingClientRect`
-  // (currently only IE and Firefox), it becomes far easier to obtain
-  // true offsets.
-  if ('getBoundingClientRect' in document.documentElement) {
```

Comment:
It's hard to tell from the diff but it's the only `viewportOffset` function I use (so not getting rid of it), it's the one used.